pipeline/runtime/ci_nudge.go

```diff
@@ -0,0 +1,16 @@
+package runtime
+
+import (
+	"errors"
+	"github.com/harness/lite-engine/logstream"
+)
+
+func getCINudges() []logstream.Nudge {
+	return []logstream.Nudge{
+		logstream.NewNudge("[Kk]illed", "Increase memory resources for the step", errors.New("out of memory")),
```

Comment:
Can we provide some doc around this?

Reply:
This is a CI problem it's not one of our messages